KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -181,6 +182,35 @@ private void loadAssetParams() {
 			impersonateClient = null;
 		}
 
+		public synchronized void setReadyForPlayback() {
+			if (readyForPlayback) {
+				logger.debug("Attention - READY_FOR_PLAYBACK already raised");
+				return;
+			}
+			if (!registered) {
```

Comment:
registered is a bad name. had to read more code& comments to know WHAT is registered